ob-jupyter.el

```diff
@@ -834,10 +834,11 @@ mapped to their appropriate minted language in
 (defun org-babel-jupyter-make-local-aliases ()
   (let ((default-directory user-emacs-directory))
     (org-babel-jupyter-aliases-from-kernelspecs)))
-(add-hook 'org-mode-hook #'org-babel-jupyter-make-local-aliases 10)
 
-(add-hook 'org-export-before-processing-functions #'org-babel-jupyter-setup-export)
-(add-hook 'org-export-before-parsing-functions  #'org-babel-jupyter-strip-ansi-escapes)
+(when jupyter-org-want-integration
```

Comment:
Why should this be here? There would be no way to recognize Jupyter source blocks in the buffer if we don't make the aliases at some point and the other hooks only apply for Jupyter specific source blocks as well.